docs/user-guide/docker/building-in-docker.rst

```diff
@@ -218,29 +220,24 @@ Your actions will run on a separate server from your Rasa server. First create a
 
 You can then run the actions with the following command:
 
-    .. code-block:: bash
+    .. parsed-literal::
 
-      docker run -d \
-        -v $(pwd)/actions:/app/actions \
-        --net my-project \
-        --name action-server
-        rasa/rasa-sdk:latest
+      docker run -d -v $(pwd)/actions:/app/actions --net my-project --name action-server rasa/rasa-sdk:\ |version|.0
```

Comment:
because you are not guaranteed to have an SDK version the same as the rasa version. e.g. there is rasa 1.8.2 but only SDK 1.8.1. The safest way to not screw it up is to always recommend (1.8).0

Reply:
to clarify, in the rasa docs, `|version|` is the minor version, while `|release|` is the patch version

Reply:
ahhh ok